src/payment/data/sagas.js

```diff
@@ -143,6 +149,10 @@ export function* handleCaptureKeyTimeout() {
   yield put(fetchCaptureKey());
 }
 
+/**
+ * Redux Saga for getting the capture context for a cybersource payment
+ * @returns
```

Comment:
Intentional?

Reply:
Yes. removed @returns since there is no return though